cedar-testing/src/integration_testing.rs

```diff
@@ -86,7 +87,7 @@ pub struct JsonRequest {
     /// of JSON value
     pub context: JsonValueWithNoDuplicateKeys,
     /// Whether to enable request validation for this request
-    #[serde(default = "constant_true")]
+    #[serde(default)]
```

Comment:
instead of changing how we interpret the JSON format, could we still parse `validate_request` correctly but then ignore it in the actual code?